shared/quantized_layer.py

```diff
@@ -0,0 +1,22 @@
+import torch
+
+
+class QuantizedLinearLayer(torch.nn.Module):
+    def __init__(self, original_layer):
+        super(QuantizedLinearLayer, self).__init__()
+        self.quantized_layer = torch.quantization.QuantStub()
+        self.dequantized_layer = torch.quantization.DeQuantStub()
+        self.original_layer = original_layer
+
+    def forward(self, x):
+        x = self.quantized_layer(x)
+        x = self.original_layer(x)
+        x = self.dequantized_layer(x)
+        return x
+
+
+# Quantize the full connection layers
+def replace_layer(model):
```

Comment:
function docstring is
```python
def replace_layer(model):
    """func docstring""""
```
see https://google.github.io/styleguide/pyguide.html#383-functions-and-methods
Also, you missed the type annotation